packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -427,18 +425,11 @@ export const ForumEventPoll = ({postId, hideViewResults, classes}: {
       if (delta) {
         voteData.delta = delta
         setCurrentUserVote(votePos)
-        if (postId) {
-          await addVote({variables: {
-            ...voteData,
-            postIds: [postId]
-          }})
-          refetch?.()
-        } else if (event.tag) {
-          openDialog({
-            componentName: "ForumEventPostSelectionDialog",
-            componentProps: {tag: event.tag, voteData}
-          })
-        }
+        await addVote({variables: {
+          ...voteData,
+          ...(postId && {postIds: [postId]}),
+        }})
```

Comment:
We'll still record the postId if the vote was changed on the page of a specific post, but this info won't be displayed anywhere